vaadin-map-flow-parent/vaadin-map-flow/src/main/java/com/vaadin/flow/component/map/configuration/Coordinate.java

```diff
@@ -19,10 +19,9 @@
 import java.util.Objects;
 
 /**
- * Represents map coordinates in a specific projection. Which projection the
- * coordinates are in is not known by the coordinate itself, and developers must
- * ensure themselves to use the same projection between coordinates, the map
- * viewport, and map sources.
+ * Represents map coordinates in a specific projection. Coordinates must be
+ * specified in the map's user projection, which by default is
+ * {@code EPSG:4326}, also referred to as GPS coordinates.
  */
 public class Coordinate {
```

Comment:
This class also has a `fromLonLat` method that needs changes, but I'm going to address that in a follow-up PR.